dotnet/src/Functions/Functions.OpenApi/Model/RestApiOperationServer.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi;
+
+/// <summary>
+/// REST API Operation Server.
+/// </summary>
+public sealed class RestApiOperationServer
+{
+    /// <summary>
+    /// REQUIRED. A URL to the target host. This URL supports Server Variables and MAY be relative,
+    /// to indicate that the host location is relative to the location where the OpenAPI document is being served.
+    /// Variable substitutions will be made when a variable is named in {brackets}.
+    /// </summary>
+#pragma warning disable CA1056 // URI-like properties should not be strings
+    public string? Url { get; }
```

Comment:
If it's required, should it be non-nullable in this case and non-optional in constructor?

Reply:
It's not required, we allow the server url to be provided via options.